dashboard/src/shared/types.ts

```diff
@@ -383,6 +383,10 @@ export interface IBasicFormParam {
   title?: string;
   minimum?: number;
   maximum?: number;
+  render?: string;
+  sliderMin?: number;
+  sliderMax?: number;
```

Comment:
I tried that but the problem is that if a parameter contains `minimum` or `maximum`, it has precedence over the type. This means that it will always fail because it expect the field to be a `integer` instead of a string. Also the meaning is a bit different. This value is used to set the min and max values for the slider but you are actually allowed to write a bigger number if you want in the input box.